web/frontend/src/components/board/index.svelte

```diff
@@ -237,31 +251,61 @@ const handleAnalogValueInput = (event:CustomEvent<{ value: string}>) => {
         </td>
       </tr>
     </table>
+
     <h3 class="mb-2">
-      Analog Write
+      Analogs
      </h3>
      <table class="mb-4 w-full table-auto border border-medium">
-     <td class="border border-medium p-2">
+      <tr>
+        <th class="border border-medium p-2">
+          Get
+        </th>
+        <td class="p-2">
+          <div class="flex flex-wrap items-end gap-2">
+               <v-input
+                 label="Pin"
+                 type="text"
+                 value={analogPinName}
+                 on:input={handleReadAnalogPinInput}
+               />
+             <v-button
+             class="mr-2"
+             label="Get Analog Value"
+             on:click={readAnalog}
+           />
+           <span class="py-2">
+            {readAnalogMessage}
+            </span>
+          </div>
+        </td>
+      </tr>
+      <tr>
+     <th class="border border-medium p-2">
+          Set
+        </th>
+       <td class="border border-medium p-2">
```

Comment:
Great job!!! 
Looks like the Analogs box at the top is now redundant. Let's remove it in this PR.

Reply:
talked with rand offline, going to keep for now.